symfony/messenger/4.3/config/packages/messenger.yaml

```diff
@@ -0,0 +1,13 @@
+framework:
+    messenger:
+        # Uncomment the following line and choose a transports for collect all messages failed and try it later
+        # failure_transport: ...
+
+        transports:
+            # Uncomment the following lines to enable one of the transports named "amqp" or "sync"
+            # amqp: '%env(MESSENGER_TRANSPORT_DSN)%'
```

Comment:
What about calling this `async`? It's not related to amqp anymore. What I mean is: amqp is now just *one* option for your transport

Reply:
I like it, we will also have to update the documentation about it